### PR TITLE
add newhope to pqdb

### PR DESCRIPTION
**Pull request by jvdheyden**

### CODE REVIEW

File: encryption/newhope/ccakem/impl/ref.yaml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+name: Reference
+platform: c
+type: reference
+side channel guarding:
+  branching: false
+  timing comment: ''
```
Comment (dingens) [inline encryption/newhope/ccakem/impl/ref.yaml]: same

**Comment by dingens** [comment]
also "request changes" nicht wegen dem kleinkram, aber wegen dem pqm4

File: encryption/newhope/ccakem/bench/ref_1024_cortex_m4.yaml [hunk 2]
```diff
@@ -0,0 +1,8 @@
+impl: cortex_m4
```
[the remark this answers is not in the document]
Comment (jvdheyden) [reply encryption/newhope/ccakem/bench/ref_1024_cortex_m4.yaml]: So I rename the file ref_1024_pqm4.yaml and call the impl pqm4?

[the remark this answers is not in the document]
Comment (dingens) [reply encryption/newhope/ccakem/bench/ref_1024_cortex_m4.yaml]: I found this description of the different implementations.  https://github.com/mupq/pqm4#schemes-included-in-pqm4

In the NewHope spec they say "For the performance estimation of NewHope on ARM Cortex-M4 micrcontrollers we refer to “m4” implementation of NewHope1024-CCA-KEM included in pqm4"

**Pull request merged**